src/index.d.ts

```diff
@@ -0,0 +1,6 @@
+export const alphabetize: (sentenceArray: string[], type: 'english' | 'unicode') =>  string;
+export const  ascii: (letters: string) => string ;
+export const firstLetters: (words: string, eng?:boolean, simplify?: boolean) => string;
+export const mainLetters: (words: string, simplify?: boolean, simplifyConsonants?: boolean) =>  string;
+export const translit: (gurmukhi: string) =>  string;
```

Comment:
also takes language and map https://github.com/KhalisFoundation/anvaad-js/blob/master/src/translit.js#L22
```
export const translit: (gurmukhi: string, language?: 'english' | 'devnagri', 'ipa', 'shahmukhi', map?: null | Map<string, string> ) =>  string;
```